documentation/docs/Exercises/Exercise 4.md

```diff
@@ -59,95 +57,158 @@ source .profile
 
 [moniker] can be any name you like
 ```bash
-gaiad init [moniker]
+terrad init [moniker]
 ```
-5. Use any text editor to open `$HOME/.gaia/config/client.toml`, edit `chain-id` and `node` fields
+5. Use any text editor to open `$HOME/.terra/config/client.toml`, edit `chain-id` and `node` fields
 ```bash
-chain-id = "cosmoshub-testnet"
-node = "https://rpc.testnet.cosmos.network:443"
+chain-id = "bombay-12"
+node = "tcp://adc1043f1d76249009c417dcad0bc807-1055950820.us-east-2.elb.amazonaws.com:26657"
 ```
 Verify you have access to the testnet, you will see the latest block info
 ```bash
-gaiad q block
+terrad status
 ```
-5. Create a key pair
+6. Create a key pair
 
-[cosmos-key-name] can be any name you like
+[terra-key-name] can be any name you like
 ```bash
-gaiad keys add [cosmos-key-name]
+terrad keys add [terra-key-name]
 ```
-6. Request tokens from the faucet
+7. Request tokens from the faucet
+Go to terra testnet faucet and get some UST for your newly creaated address https://faucet.terra.money/
+
+Verify that tokens have arrived
+
+[address] is the address you created in step 6, associated with the [terra-key-name]
 ```bash
-curl -X POST -d '{"address": "your newly created address"}' https://faucet.testnet.cosmos.network
+terrad q bank balances [address]
 ```
-When the tokens are sent, you will see the following response:
-```json
-{"transfers":[{"coin":"100000000uphoton","status":"ok"}]}
+
+## Instructions to send UST from Terra testnet to EVM compatible chains
+The flow works for any EVM compatible chains that Axelar supports. We use Ethereum Ropsten as example.
+
+1. On a new terminal window, enter Axelar node:
+```bash
+docker exec -it axelar-core sh
 ```
-Check that tokens have arrived
 
-[cosmoshub address] is the address you created in step 5, associated with the [cosmos-key-name]
+2. Create a deposit address on Axelar Network (to which you'll deposit coins later)
+[receipent addres] is the address you want to receive the tokens to
+```bash
+axelard tx axelarnet link [evm chain] [receipent address] uusd --from [axelar-key-name]
+```
+e.g.,
 ```bash
-gaiad q bank balances [cosmoshub address]
+axelard tx axelarnet link ethereum 0x4c14944e080FbE711D29D5B261F14fE4E754f939 uusd --from validator
 ```
-### Instructions to send tokens from Cosmoshub testnet to Axelar Network
-1. Send an IBC transfer from Cosmoshub testnet to Axelar Network
+Look for `successfully linked [Axelar Network deposit address] and [receipent address]`
 
-You can find `Cosmoshub channel id` under [Testnet Release](/testnet-releases)
+3. Send an IBC transfer from Terra testnet to Axelar Network
+Switch back to terminal with terrad installed
 
-[axelar address] is the address you generated in Exercise 3
+```bash
+terrad tx ibc-transfer transfer transfer [Terra channel id] [Axelar Network deposit address] --packet-timeout-timestamp 0 [amount]uusd --gas-prices 0.15uusd --from [terra-key-name] -y -b block
```

Comment:
duplicate `transfer`??

Reply:
na, the first transfer is command name, the second transfer is port id